pkgs/applications/misc/kepubify/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "kepubify";
+  version = "2.3.3";
+  rev = "v${version}";
```

Comment:
Is there a reason rev is here?

Reply:
Nope, moved into the `src = fetchFromGithub {` block.